client/src/components/App.jsx

```diff
@@ -22,9 +26,29 @@ class App extends React.Component {
     });
   }
 
+  handleLeftArrowClick() {
+    const { currentPhoto, photos } = this.state;
+    const newCurrentPhoto =
+      currentPhoto > 0 ? currentPhoto - 1 : photos.length - 1;
+
+    this.setState({
+      currentPhoto: newCurrentPhoto,
+    });
+  }
+
+  handleRightArrowClick() {
+    const { currentPhoto, photos } = this.state;
+    const newCurrentPhoto =
+      currentPhoto < photos.length - 1 ? currentPhoto + 1 : 0;
+
+    this.setState({
+      currentPhoto: newCurrentPhoto,
+    });
+  }
+
```

Comment:
This could be DRYed up and refactored into a single method if you have the time. Looks functional otherwise.